sphinx/api/built-in/rose_bunch.rst

```diff
@@ -49,28 +49,33 @@ Configuration
 
 The application is normally configured in the
 :rose:conf:`rose_bunch[bunch]` and :rose:conf:`rose_bunch[bunch-args]`
-sections of the :rose:file:`rose-app.conf` file.
+sections of the :rose:file:`rose-app.conf` file, but `rose_bunch[command]`
```

Comment:
We use the `:rose:conf:` role to link the setting back to the API reference. This way we can detect broken references etc.

```rst
:rose:conf:`rose_bunch[command]`
```

Reply:
Actually, I think this should be
```
:rose:conf:`rose-app.conf[command]`
```
I will tweak the docs.

Reply:
Yes!